std/regex/package.d

```diff
@@ -681,18 +627,6 @@ public:
 
     ///A hook for compatibility with original std.regex.
     @property ref captures(){ return this; }
-
-    typeof(this) opAssign()(auto ref Captures rhs)
```

Comment:
don't you want to manually overload this to copy the Store?

Reply:
Why would I - store is copied + bitblitter by default, no?